### PR TITLE
Refactor based on Paraspell V2 changes

### LINKED ISSUE #46: Refactor

### PR DESCRIPTION
**Pull request by brano-hozza**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Fix
- [ ] Feature
- [x] Refactoring
- [ ] Chore

### What's new?

- [x] PR closes #46 
- [x] Lots of changed files

### Before submitting Pull Request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR
- [ ] I've written some unit tests 🧪

### Optional

- [ ] I found edge cases

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### CODE REVIEW

File: app.config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default defineAppConfig({
-  title: 'B App',
+  title: 'Paraspell',
```
Comment (vikiival) [inline app.config.ts:2]: 👀

**Review by vikiival — COMMENTED** [review]
Smol
